ui/src/App.vue

```diff
@@ -25,24 +27,39 @@ export default {
   },
   data() {
     return {
-      isRemote: {},
-      employmentTypes: {},
-      locations: {},
-      skills: {}
+      isRemote: [],
+      employmentTypes: [],
+      locations: [],
+      skills: []
     }
   },
-  methods: {
-    onSubmit() {
-      axios.get(mockAPI)
+  mounted() {
+    axios.get(mockAPI)
         .then(response => {
-          console.log(response);
-          // this.isRemote = response.isRemote;
+          this.isRemote = response.data.is_remote.buckets;
+          this.employmentTypes = response.data.employment_type.buckets;
+          this.locations = response.data.location.buckets;
+          this.skills = response.data.skills.buckets;
```

Comment:
Should pass the data received from backend to a transformation/mapper function. This code is weird.